orttraining/orttraining/lazy_tensor/accelerator.cpp

```diff
@@ -0,0 +1,250 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "accelerator.h"
+#include <iostream>
+#include <string>
+#include <torch/csrc/jit/passes/onnx.h>
+#include <torch/torch.h>
+#include "bridge.h"
+#include "core/common/logging/sinks/clog_sink.h"
+#include "core/framework/session_options.h"
+#include "core/session/environment.h"
+#include "python/onnxruntime_pybind_state_common.h"
+
+namespace onnxruntime {
+namespace lazytensor {
+
+namespace py = pybind11;
+namespace aten = torch::jit::aten;
+namespace prim = torch::jit::prim;
+
+// static variable used to create inference session and training session.
+const static std::string env_name = std::string("LTC");
+static std::unique_ptr<onnxruntime::Environment> ltc_env;
+
+onnxruntime::Environment& GetLtcEnv() {
+  if (!ltc_env) {
+    ORT_THROW_IF_ERROR(
+        onnxruntime::Environment::Create(
+            std::make_unique<onnxruntime::logging::LoggingManager>(
+                std::make_unique<onnxruntime::logging::CLogSink>(),
+                onnxruntime::logging::Severity::kWARNING,
+                false,
+                onnxruntime::logging::LoggingManager::InstanceType::Temporal,
+                &env_name),
+            ltc_env));
+  }
+  return *ltc_env;
+}
+
+bool Accelerator::Supported(const torch::jit::Node* node) {
+  if (!node) {
+    return false;
+  }
+
+  switch (node->kind()) {
+    //case aten::relu:
+    case aten::mul:
+    case aten::add:
+    case aten::sub:
+    case aten::div:
+    //case aten::gt:
+    //case aten::eq:
+    case prim::Constant:
+      //case aten::threshold_backward:
+      std::cout << "[compiler.cc] Support " << *node;  //<< std::endl;
```

Comment:
[accelerator.cpp]? No logging instead of std::cout?

Reply:
I'd like to decouple the logging from ORT for now so we can print more focusing information.